examples/stress_tests/README.md

```diff
@@ -1,3 +1,10 @@
 # Stress tests
 
-These examples are used to stress test Bevy's performance in various ways. These should be run with the --release argument to cargo or equivalent optimization, otherwise they will be very slow.
+These examples are used to stress test Bevy's performance in various ways. 
+These should be run with the "stress-test" profile, otherwise they will be very slow or inaccurately represent production level performance.
```

Comment:
```suggestion
These should be run with the "stress-test" profile to accurately represent performance in production, otherwise they will run in cargo's default "dev" profile which is very slow.
```